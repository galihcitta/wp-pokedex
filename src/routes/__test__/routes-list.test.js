import routesList from "../routes-list";

describe("test app", () => {
    it("routesList snapshoot", () => {
        expect(routesList).toMatchSnapshot();
    });
});
