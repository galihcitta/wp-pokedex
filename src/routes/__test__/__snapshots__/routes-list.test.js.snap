// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test app routesList snapshoot 1`] = `
Array [
  Object {
    "component": Object {
      "$$typeof": Symbol(react.lazy),
      "_ctor": [Function],
      "_result": null,
      "_status": -1,
    },
    "path": "/",
    "role": Array [
      "GUEST",
    ],
  },
  Object {
    "component": Object {
      "$$typeof": Symbol(react.lazy),
      "_ctor": [Function],
      "_result": null,
      "_status": -1,
    },
    "path": "/pokemon-detail/:slug",
    "role": Array [
      "GUEST",
    ],
    "style": Object {
      "header": Object {
        "withBackButton": true,
      },
    },
  },
  Object {
    "component": Object {
      "$$typeof": Symbol(react.lazy),
      "_ctor": [Function],
      "_result": null,
      "_status": -1,
    },
    "path": "/404",
    "role": Array [
      "USER",
      "GUEST",
    ],
  },
]
`;
